<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlPlayBack</class>
 <widget class="QMainWindow" name="SqlPlayBack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SqlPlayBack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonConnectDB">
            <property name="text">
             <string>Connect To DBase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveToCSV">
            <property name="text">
             <string>Save to CSVs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStopPlay">
            <property name="text">
             <string>StopPlayback</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPlayToMultiverse">
            <property name="text">
             <string>PlayBack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFetchFromDB">
            <property name="text">
             <string>Fetch From DBase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFetchFromCSV">
            <property name="text">
             <string>Fetch From CSVs</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>338</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>StartTime</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEditStart">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>End</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEditEnd">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNbrLast">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetLastValues"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>MS Playback</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPlayBackSpeed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditTargetCsv">
            <property name="minimumSize">
             <size>
              <width>275</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetTarget">
            <property name="minimumSize">
             <size>
              <width>275</width>
              <height>400</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTargetTag">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditPartnerCsv">
            <property name="minimumSize">
             <size>
              <width>275</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetPartner">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPartnerTag">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditStatusMessages"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Files">
    <property name="title">
     <string>&amp;Files</string>
    </property>
    <addaction name="action_Sql"/>
    <addaction name="action_Multiverse"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_Files"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_MySql">
   <property name="text">
    <string>&amp;MySql</string>
   </property>
  </action>
  <action name="action_PostGres">
   <property name="text">
    <string>&amp;PostGres</string>
   </property>
  </action>
  <action name="action_Multiverse">
   <property name="text">
    <string>Multi&amp;Verse</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_StartDate">
   <property name="text">
    <string>&amp;StartDate</string>
   </property>
  </action>
  <action name="action_Back_Hours">
   <property name="text">
    <string>&amp;Back Hours</string>
   </property>
  </action>
  <action name="action_Select">
   <property name="text">
    <string>&amp;Select</string>
   </property>
  </action>
  <action name="action_Begin">
   <property name="text">
    <string>&amp;Begin</string>
   </property>
  </action>
  <action name="action_Select_2">
   <property name="text">
    <string>&amp;Select</string>
   </property>
  </action>
  <action name="action_Sql">
   <property name="text">
    <string>&amp;Sql</string>
   </property>
  </action>
  <action name="actionTarget_CSF">
   <property name="text">
    <string>Target CSV</string>
   </property>
  </action>
  <action name="actionPartner_CSV">
   <property name="text">
    <string>Partner CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
